ui/desktop/app/templates/application.hbs

```diff
@@ -21,30 +21,35 @@
 
       <header.utilities as |utilities|>
         {{#if this.showWindowActions}}
-          <Hds::Button
-            @icon='x'
-            @size='small'
-            @text={{t 'actions.close'}}
-            @color='critical'
-            class='button-window-close'
-            {{on 'click' this.close}}
-          />
-          <Hds::Button
-            @icon='x'
-            @size='small'
-            @color='secondary'
-            @text={{t 'actions.fullscreen'}}
-            class='button-window-fullscreen'
-            {{on 'click' this.toggleFullScreen}}
-          />
-          <Hds::Button
-            @icon='x'
-            @size='small'
-            @color='secondary'
-            @text={{t 'actions.minimize'}}
-            class='button-window-minimize'
-            {{on 'click' this.minimize}}
-          />
+          <Hds::ButtonSet>
```

Comment:
Here's a demo of what the updated UI looks like:
<img width="1332" alt="Screenshot 2025-04-15 at 6 30 21 PM" src="https://github.com/user-attachments/assets/bc9e9120-6c36-41b3-8ffe-3d873ef3200e" />